cli/src/main.ts

```diff
@@ -50,8 +51,10 @@ module.exports = function (argv: string[]): void {
                 console.warn("Ignoring option '--baseImagesUrl' for prepackaged extension.");
             if (extensionFile !== undefined && yarn !== undefined)
                 console.warn("Ignoring option '--yarn' for prepackaged extension.");
+            if (extensionFile !== undefined && web !== undefined)
+                console.warn("Ignoring option '--web' for prepackaged extension.");
```

Comment:
Is it true that the option is ignored in this case? The server can handle the flag anyway, right?

Reply:
That's right. The flag is passed to the server and the files are processed there anyway. The warning is confusing, I will remove it.